src/main/java/fuel/domain/RentCompany.java

```diff
@@ -0,0 +1,27 @@
+package fuel.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RentCompany {
+    private static final String NEWLINE = System.getProperty("line.separator");
+
+    private final List<Car> carLists = new ArrayList<>();
```

Comment:
`List`라는 특정 Collection명은 제거해주는게 어떨까요? `cars`로도 의미가 충분해보여요!

```suggestion
    private final List<Car> cars = new ArrayList<>();
```